weather_aplication/GeoDBJSON.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+import SnapKit
+
+class GeoDBJSON: Decodable {
+    let data: [CityInfo]
+    let links: [Link]?
+    let metadata: Metadata
+
+    init(data: [CityInfo], links: [Link]?, metadata: Metadata) {
+        self.data = data
+        self.links = links
+        self.metadata = metadata
+    }
+}
+
+class CityInfo: Decodable {
+    let id: Int?
+    let wikiDataID, type, city, name: String?
+    let country, countryCode, region, regionCode: String?
```

Comment:
raczej nie robimy takiego czegoś, znacznie bardziej czytelne jest robienie osobnych lotów. Poza tym nawet jakbyś chciał zrobić po przecinku to tutaj masz 2 linijki mimo że typ jest ten sam